bodhi/server/models/models.py

```diff
@@ -1260,11 +1264,26 @@ def modify_bugs(self):
                     # https://github.com/fedora-infra/bodhi/issues/368#issuecomment-135155215
                     for bug in self.bugs:
                         if not bug.parent:
-                            log.debug("Closing tracker bug %d" % bug.bug_id)
-                            bug.close_bug(self)
+                            bad, good = self.get_bug_karma(bug)
+                            bug_karma = good + bad
+                            if bug_karma >= 0:
+                                log.debug("Closing tracker bug %d" % bug.bug_id)
+                                bug.close_bug(self)
+                            else:
+                                log.debug('Adding stable comment to bugs for %s', self.title)
+                                bug.add_comment(self)
```

Comment:
I felt that adding a text was a daunting job. the comment is added to the bug based on the update state, but also handled at the end by the masher with fedmsg, and needed to add that text to the *.ini files.

I don't know how fedmsg works and how to pass a certain value for its context to be able to decide if this text will be added to this bug or not.